docs/tutorial.rst

```diff
@@ -102,11 +104,415 @@ mlf-core is now configured and ready to be used!
+
+.. code::
+
+    ├── .bandit.yml <- Configuration file for Bandit (identifies security issues in the code)
+    ├── CHANGELOG.rst <- Changelog of the project (controlled by mlf-core bump-version)
+    ├── CODE_OF_CONDUCT.rst
+    ├── Dockerfile <- Dockerfile specifying how the Docker container is build; Uses the environment.yml file to create a Conda environment inside the container
+    ├── docs
+    │   ├── authors.rst
+    │   ├── changelog.rst
+    │   ├── code_of_conduct.rst
+    │   ├── conf.py <- Sphinx configuration file
+    │   ├── index.rst <- Root of the documentation; defines the toctree
+    │   ├── make.bat <- Windows version of the Makefile
+    │   ├── Makefile <- Makefile for the documentation (run   make html   to build the documentation)
+    │   ├── model.rst <- Model documentation
+    │   ├── readme.rst
+    │   ├── requirements.txt <- Defines Python dependencies for the documentation
+    │   ├── _static
+    │   │   └── custom_cookietemple.css <- Custom dark documentation style
+    │   └── usage.rst <- How to use the mlf-core model
+    ├── .editorconfig <- Configuration for IDEs and editors
+    ├── environment.yml <- Defines all dependencies for your project; Used to create a Conda environment inside the Docker container
+    ├── project_name
+    │   ├── data_loading
+    │   │   ├── data_loader.py <- Loading and preprocess training/testing data
+    │   ├── mlf_core
+    │   │   └── mlf_core.py <- mlf-core internal code to run system-intelligence and advanced logging; Should usually not be modified
+    │   ├── model
+    │   │   ├── model.py <- Model architecture
+    │   ├── project_name.py <- Entry point for MLflow; Connects all pieces
+    ├── .flake8 <- flake8 configuration file (lints code style)
+    ├── .gitattributes <- git configuration file
+    ├── .github
+    │   ├── ISSUE_TEMPLATE
+    │   │   ├── bug_report.md
+    │   │   ├── feature_request.md
+    │   │   └── general_question.md
+    │   ├── pull_request_template.md
+    │   └── workflows
+    │       ├── lint.yml <- Runs mlf-core lint and flake8 on push events
+    │       ├── master_branch_protection.yml <- Protects the master branch from non-release merges
+    │       ├── publish_docker.yml <- Publishes the Docker container on Github Packages (or alternatives)
+    │       ├── publish_docs.yml <- Publishes the documentation on Github Pages or Read the Docs
+    │       ├── sync.yml <- Checks for new mlf-core templates versions and triggers a PR with changes if found; Runs daily
+    │       └── train_cpu.yml <- Trains the model with a reduced dataset on the CPU
+    ├── .gitignore
+    ├── LICENSE
+    ├── mlf_core.cfg <- mlf-core configuration file (sync, bump-version, linting, ...)
+    ├── .mlf_core.yml <- Meta information of the mlf_core.yml file; Do not edit!
+    ├── MLproject <- MLflow Project file; Defines entry point and parameters
+    ├── README.rst
+    └── .readthedocs.yml <- Read the Docs configuration file
```

Comment:
I really like this tree overview, though I have one question/suggestion: is it possible to somehow colorize the comments for each file? Would improve readibility a lot (maybe think about creating a rich tree view and screenshot it?)

Reply:
Yeah I wanted to do so but didn't find a way to use custom colors in a code directive.
Screenshots could work but are hard to maintain.